package-lock.json

```diff
@@ -2258,7 +2258,8 @@
         "ansi-regex": {
```

Comment:
Please do not change this file like this

Reply:
Package lock gets updated by running npm install, these changes should be committed. I see nothing wrong with this.